LocalSettings.php

```diff
@@ -3241,8 +3241,7 @@
 			'monobook',
 			'vector',
 			'timeless',
-			'minerva',
-			'minervaneue'
```

Comment:
Did you intend to remove a skin here?

Reply:
Yes, keeping minerva, removing minervaneue, as minervaneue does nothing